pkgs/development/libraries/usbredir/default.nix

```diff
@@ -12,14 +12,13 @@ stdenv.mkDerivation rec {
   NIX_CFLAGS_COMPILE = "-Wno-error";
 
   nativeBuildInputs = [ pkgconfig ];
-  buildInputs = [ libusb ];
-  propagatedBuildInputs = [ libusb ];
```

Comment:
Looks like `$dev/include/usbredirhost.h` contains `#include <libusb.h>` so this might be still necessary but the `.pc` file is incorrect.

Reply:
but shouldn't we still explicitly configure these dependencies where required? With `propagatedBuildInputs` being set, we didn't notice we were missing it from `spice-gtk`…

Reply:
I added `libusb1` back to `usbredir`'s `propagatedBuildInputs`.